templates/zerver/help/set-notifications-for-a-single-stream.md

```diff
@@ -0,0 +1,22 @@
+# Set notifications for a single stream
+
+On Zulip, you can have different notification settings for different streams.
+
+1. To set notifications for a single stream, find the **Streams**
+ list located on the left sidebar.
+
+2. On the **Streams** list, hover over the stream that you want to set
+ notifications for to reveal a down chevron
+ (<i class="icon-vector-chevron-down"></i>).
```

Comment:
@synicalsyntax is currently working on changing all cases of the image to the actual icon. For example, the icon is used in the macro for down-chevrons at templates/server/help/include/down-chevron.md